api/src/certification/results/infrastructure/utils/pdf/AttestationViewModel.js

```diff
@@ -58,9 +81,12 @@ class AttestationViewModel {
   }
 
   shouldDisplayProfessionalizingCertificationMessage() {
-    if (!this._isFrenchDomainExtension) return false;
     if (!this._deliveredAt) return false;
-    return this._deliveredAt.getTime() >= PROFESSIONALIZING_VALIDITY_START_DATE.getTime();
+    return (
+      this._isFrenchDomainExtension &&
+      this._deliveredAt.getTime() >= PROFESSIONALIZING_VALIDITY_START_DATE.getTime() &&
```

Comment:
on peut extraire `this._deliveredAt.getTime() >= PROFESSIONALIZING_VALIDITY_START_DATE.getTime()`
dans une fonction dédiée pour exprimer la règle

Reply:
ok !